lib/log_ratelimit.c

```diff
@@ -31,6 +33,8 @@ struct log_ratelimit_state {
 	uint32_t printed;
 	uint32_t suppressed;
 	uint64_t end;
+	uint32_t log_level;
```

Comment:
I know that memory is aligned and `uint32_t` is less than 64 bits, but we should use `rte_atomic32_t` here to avoid implicit assumptions about synchronization. The new type will not add overhead.